### PR TITLE
demo: add tooltip disabled demo

### PR DESCRIPTION
**Pull request by JarvisArt**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | add Tooltip disabled demo |
| 🇨🇳 Chinese | 添加 Tooltip 禁用 demo |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at be64547</samp>

This pull request adds a new demo `disabled-children.tsx` to the tooltip component, which shows how to disable the tooltip for the wrapped children elements. It also updates the existing demo `disabled.tsx` and its documentation to clarify the difference between disabling the tooltip itself and disabling the tooltip for the wrapped children elements.

### <samp>🤖 Generated by Copilot at be64547</samp>

*  Add a new demo `disabled-children.tsx` and its documentation to show how to disable the tooltip for the wrapped children elements using the `WrapperTooltip` component ([link](https://github.com/ant-design/ant-design/pull/45748/files?diff=unified&w=0#diff-b80300d0e77a0e5a5da825a475fd80c78e7e57d8933b395046a6cd3063476278R1-R7), [link](https://github.com/ant-design/ant-design/pull/45748/files?diff=unified&w=0#diff-eb83bb3c97c0c200adc08d7ded9614c983adf10132ef39ce7f51ef3de6107a1aR1-R28), [link](https://github.com/ant-design/ant-design/pull/45748/files?diff=unified&w=0#diff-5e2180bb3595a48a443b9845ce81890929e5b99580df55b3905dda6805484876L30-R31), [link](https://github.com/ant-design/ant-design/pull/45748/files?diff=unified&w=0#diff-55ca4dc68bad04b61d30f286605588ea4eb77dea56527d9bfbfbeada5011e4c1L32-R33))
*  Modify the existing demo `disabled.tsx` and its documentation to clarify that it shows how to disable the tooltip itself, not the wrapped children elements, and to simplify the code using a button and a span ([link](https://github.com/ant-design/ant-design/pull/45748/files?diff=unified&w=0#diff-b296dfa0ce3774322344e8e9c8db42e5701d7ffc76148d2ee7bec691ca1daa37L3-R7), [link](https://github.com/ant-design/ant-design/pull/45748/files?diff=unified&w=0#diff-308565e8e1b5623df5629ddc3caca033c893b083e81a1f385824537ee6dd52f0L1-R15), [link](https://github.com/ant-design/ant-design/pull/45748/files?diff=unified&w=0#diff-5e2180bb3595a48a443b9845ce81890929e5b99580df55b3905dda6805484876L30-R31), [link](https://github.com/ant-design/ant-design/pull/45748/files?diff=unified&w=0#diff-55ca4dc68bad04b61d30f286605588ea4eb77dea56527d9bfbfbeada5011e4c1L32-R33))

### CODE REVIEW

File: components/tooltip/demo/disabled.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## zh-CN
 
-Disabled 子元素。
+禁用 Tooltip
```
Comment (afc163) [inline components/tooltip/demo/disabled.md:3]:
```suggestion
通过设置 `title=""` 可以禁用 Tooltip。
```

File: components/tooltip/demo/disabled.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ## zh-CN
 
-禁用 Tooltip
+通过设置 `title=""` 可以禁用 Tooltip。
 
 ## en-US
 
-Disabled Tooltip.
+The Tooltip can be disabled by setting title="".
```
Comment (afc163) [inline components/tooltip/demo/disabled.md:7]:
```suggestion
The Tooltip can be disabled by setting `title=""`.
```

Comment (afc163) [inline components/tooltip/demo/disabled.md:7]:
```suggestion
The Tooltip can be disabled by setting `title=""`.
```

Comment (JarvisArt) [reply components/tooltip/demo/disabled.md]: done~

**Review by afc163 — APPROVED** [review]

**Pull request merged**